druid-shell/src/mouse.rs

```diff
@@ -21,9 +21,7 @@ use crate::keyboard::KeyModifiers;
 /// Information about the mouse event.
 #[derive(Debug, Clone, PartialEq)]
 pub struct MouseEvent {
-    /// The location of the mouse in the current window.
-    ///
-    /// This is in px units not device pixels, that is, adjusted for hi-dpi.
+    /// The location of the mouse in display points in relation to the current window.
```

Comment:
I would include a section in the docs of `scale.rs` that explicitly explains our use of 'pixel/px' and 'display point/dp', and then we can link to that section everywhere we use one of those terms?

It would be nice if we had some canonical doc explanation of the dp/px distinction, and then we could link to that everywhere we use one of those terms? I'd make this super obvious, something with a section in the `//! top level` doc comment called like '## Pixels (px) and display points (dp)'. Or something? I'd just like it to be easy for someone reading docs (more important in druid than druid-shell I guess) to be able to easily access an explanation.

Reply:
Sure, I'll write some more docs about it. I also already wrote a book chapter about it in #913.